### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Yadunund**
Adding a template for reporting bugs. Format open for discussion.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+**Workspace (please complete the following information):**
+ - OS:
+ - RMF binaries version: 
+ If building from source,
+ - rmf_demos branch:
+ - rmf_core branch:
+ - rmf_schedule_visualizer branch:
+ - traffic_editor branch:
+
+**To Reproduce**
```
Comment (gbiggs) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]:
```suggestion
**Steps to Reproduce**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
Comment (gbiggs) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Add a comment notice about whether or not to ask for technical support via bug reports.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ - OS:
+ - RMF binaries version: 
+ If building from source,
+ - rmf_demos branch:
+ - rmf_core branch:
+ - rmf_schedule_visualizer branch:
+ - traffic_editor branch:
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (gbiggs) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
1. 
2. 
3. ```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (gbiggs) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]:
```suggestion

**Actual behavior**
A description of what actually happens.

```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+A clear and concise description of what the bug is.
+
+**Workspace (please complete the following information):**
+ - OS:
+ - RMF binaries version: 
+ If building from source,
+ - rmf_demos branch:
+ - rmf_core branch:
+ - rmf_schedule_visualizer branch:
+ - traffic_editor branch:
```
Comment (gbiggs) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
**Workspace information:**
 - Operating system and version:
 - ROS distribution:
 - ROS installation method: [from binary packages or from source]
 - RMF installation method: [from binary packages or from source]
 - RMF binaries version, if installed from binaries:
 - RMF source versions, if installed from source:
   - `rmf_demos` branch or commit hash: 
   - `rmf_core` branch or commit hash: 
   - `rmf_schedule_visualizaer` branch or commit hash: 
   - `traffic_editor` branch or commit hash: 
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (Yadunund) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: I guess users could use the same form when submitting bug reports or asking for technical support? Unless we want to have separate sections for these?

Comment (gbiggs) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: I recommend being very careful about using GitHub's issues for technical support. We should set up a Discussions thing and point users there, I think.

Comment (Yadunund) [reply .github/ISSUE_TEMPLATE/bug_report.md:14]: Thanks for the suggestion. I've added a comment to redirect technical support enquiries to Discussions. 
https://github.com/osrf/rmf_demos/pull/184/commits/ba264e4b71a8c32f03ec6770f7c4baba7d149e1b

**Review by gbiggs — APPROVED** [review]

**Pull request merged**